docs/articles/design.R

```diff
@@ -0,0 +1,14 @@
+## ----setup, include = FALSE----------------------------------------------
```

Comment:
Why does this file exist?

Reply:
Mistake when trying out to build pkgdown.